panel/browser-panel-client.cpp

```diff
@@ -76,11 +76,12 @@ void QCefBrowserClient::OnTitleChange(CefRefPtr<CefBrowser> browser,
 		if (title.compare("DevTools") == 0)
 			return;
 
+#if defined(_WIN32)
 		CefWindowHandle handl = browser->GetHost()->GetWindowHandle();
-#ifdef _WIN32
 		std::wstring str_title = title;
 		SetWindowTextW((HWND)handl, str_title.c_str());
 #elif defined(__linux__)
+		CefWindowHandle handl = browser->GetHost()->GetWindowHandle();
```

Comment:
What is this fixing? Can it be split into its own PR?

Reply:
It fixes an "unused variable" error, because `handl` is only used on Windows and "shadowed" by the second assignment on Linux (so two violations in one).

Reply:
Given the small scope of the PR here I found it more convenient to bundle them here.

Reply:
Only used on Windows and Linux, leaving macOS the odd one out with the error, I think?

Reply:
Either way, the change itself seems fine. I just like to try to separate the code changes from the CMake changes as much as possible to minimize review burdens and scope.